packages/odyssey-react-mui/src/labs/DataView/DataView.test.tsx

```diff
@@ -960,10 +960,10 @@ describe("DataView", () => {
     const rows = await screen.findAllByRole("row");
     const tHead = rows[0].parentElement;
 
-    // Ensure that the resize handle is displayed when
-    // hasColumnResizing is true
-    const hrElement = tHead!.querySelector("hr");
-    expect(tHead).toContainElement(hrElement);
+    // Get all separators and pick the first one
+    const separators = within(tHead!).getAllByRole("separator");
```

Comment:
I wonder if we could rewrite this such that we're not using HTML child selectors on `rows` and instead use `getByRole("rowgroup", { name: "" })` to something that gets us the `thead`.